### PR TITLE
Modularise posts actions

### PR DESCRIPTION
**Pull request by sgomes**
This is a reorganisation of posts action creators, which all lived inside a single `actions.js` file with > 700 lines of code. This PR splits them into individual modules, re-exporting them from an index file. It's a follow-up to all the previous posts modularisation work.

I split each extraction into its own commit, in case that makes things easier for reviewers.

This PR *may* help with loading performance, in case there are any seldom-used albeit heavy action creators in there, as they can now move freely to just the chunks they're needed in.

#### Changes proposed in this Pull Request

* Split `state/posts/actions.js` into individual modules

#### Testing instructions

This PR only changes where things live, with no changes to functionality. As such, it should be sufficient to ensure that the build continues to work and the existing unit tests don't fail.

### CODE REVIEW

File: client/state/posts/actions/normalize-post.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+function getPageTemplate( post ) {
+	if ( ! post || ! post.page_template || post.page_template === 'default' ) {
+		return '';
+	}
+	return post.page_template;
+}
+
+export function normalizePost( post ) {
```
Comment (tyxla) [inline client/state/posts/actions/normalize-post.js]: This doesn't seem to be an action. Should we move it to some other file, utils.js or something?

**Review by tyxla — APPROVED** [review]
Looks good and tests well to me, thanks! 3KB compressed shaved off the main entry 👍 

Left a minor suggestion you might want to address optionally before merging.

File: client/state/posts/actions/normalize-post.js (hunk 1 above)
Comment (sgomes) [reply client/state/posts/actions/normalize-post.js]: Right, it's not; it's a shared util used by several actions. Renamed and moved to `utils/`.

**Review by tyxla — APPROVED** [review]
Still looking good 🚢

**Pull request merged**